package.json

```diff
@@ -1,46 +1,48 @@
 {
   "name": "siwt",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
Are we introducing a new feature that would need a version bump? Then please put v0.0.8 everywhere and we make a release after this PR

Reply:
JUST wanted to be extra sure ;)